packages/@ama-mfe/ng-utils/src/host-info/host-info.ts

```diff
@@ -47,12 +47,13 @@ export interface MFEHostInformation {
  * - use `document.referrer` (will only work if called before any redirection in the iframe)
  * The host application ID is taken from the search parameter {@link MFE_HOST_APPLICATION_ID_PARAM} in the URL of the iframe
  * The module application ID is taken from the search parameter {@link MFE_APPLICATION_ID_PARAM} in the URL of the iframe
+ * @param locationParam - optional parameter for testing purposes, defaults to `location`
```

Comment:
I would avoid to specify it is optional (as part of the type) and that it is for test purpose when it is actually used by the function on runtime.

```suggestion
 * @param locationParam - A {@link location} object with information about the current location of the document. Default to global {@link location}
```